src/core/Chip/BaseChip/BaseChip.baseStyles.tsx

```diff
@@ -21,15 +21,15 @@ export const baseChipBaseStyles = (theme: SuomifiTheme) => css`
     padding: ${theme.spacing.insetXxs} ${theme.spacing.insetL};
     color: ${theme.colors.whiteBase};
     background: ${theme.colors.highlightBase};
-    max-height: 28px;
+    max-height: 100%;
     display: inline-block;
 
     & .fi-chip--content {
-      display: inline-block;
-      max-width: 270px;
-      white-space: nowrap;
+      display: block;
+      max-width: 100%;
+      white-space: normal;
+      word-break: break-word;
       overflow: hidden;
```

Comment:
overflow: hidden; is not needed, because long string will moved to another line if its enough long.

Reply:
It is needed in cases there is no room to grow in the surrounding container or in case the chip has a static size set with styles.